include/boost/unordered/detail/foa/concurrent_table.hpp

```diff
@@ -1405,13 +1402,13 @@ class concurrent_table:
     GroupAccessMode access_mode,ExecutionPolicy&& policy,F f)const
     ->decltype(f(nullptr,0,nullptr),void())
   {
-    if(!this->arrays.elements)return;
-    auto first=this->arrays.groups,
+    if(!this->arrays.elements_)return;
```

Comment:
In general, I'd prefer to not use `elements_` directly but `elements()` instead. This happens in some other places.